CONTRIBUTING.md

```diff
@@ -1,78 +1,33 @@
 
 We aren't going to take every single little change. Don't be offended if we close your PR. In order for the project to stay healthy, we need to guard our bandwidth and also only take changes that align with Actual.
 
-Please ask first before making a large change; you might waste a lot of work if it doesn't align with how Actual should work.
-
 Here are some initial guidelines for how contributions will be treated:
 
-* The mental health of the maintainers will be prioritized above all else. If this means some things get lost and PRs are unreviewed because maintainers are spending time with family or on themselves, we celebrate that.
+- The mental health of the maintainers will be prioritized above all else. If this means some things get lost and PRs are unreviewed because maintainers are spending time with family or on themselves, we celebrate that.
 
-* Multiple maintainers are key to this being a healthy project. Currently, only I (@jlongster) have maintainer rights. I am actively looking for more people to come on as maintainers. If nobody steps up, expect less activity on this project.
+- Multiple maintainers are key to this being a healthy project. Currently a few people have maintainer rights (see list below). We are actively looking for more people to come on as maintainers. If nobody steps up, expect less activity on this project.
 
-* An open PR does not automatically deserve time for a full review and acceptance. It's up to the PR author to convince the maintainers that the change is good and worth reviewing. This involves a clear description for why the the change is being made, detailing the tradeoffs, and ideally a link to a live demo where Actual is running with the changes. (Ideally, we would automatically generate a live demo)
-
-* We especially welcome improvements in automation: creating github actions to automatically generate builds, making the release process easier, etc.
+- An open PR does not automatically deserve time for a full review and acceptance. It's up to the PR author to convince the maintainers that the change is good and worth reviewing. This involves a clear description for why the the change is being made, detailing the tradeoffs.
 
+- We especially welcome improvements in automation: creating github actions to automatically generate builds, making the release process easier, etc.
 
 ## Main contributors
 
-* @jlongster
-
-## Project ideas
-
-The code needs a lot of improvements. When it was just me (@jlongster) I would often find myself halfway through an improvement overall, but always had to put it aside to work on something else. This shows in the code; there's a lot of things that I wanted to improve but was never able to complete. These are list of projects on the top of my mind that I would love to see progress on.
```

Comment:
Since you’re removing this, do you want to make sure that all of these have a corresponding issue with the relevant information?

Reply:
Makes sense.

- mobile app: we got that already (albeit only the read only version for now)
- typescript: https://github.com/actualbudget/actual/issues/577
- table rewrite: https://github.com/actualbudget/actual/issues/655
- babel: https://github.com/actualbudget/actual/issues/656
- component lib: https://github.com/actualbudget/actual/issues/631
- custom reports: https://github.com/actualbudget/actual/issues/507